File: app/src/main/java/com/example/fittrack/Activity/IntroActivity.java
package com.example.fittrack.Activity;

import android.content.Intent;
import android.os.Bundle;
import android.view.WindowManager;

import androidx.appcompat.app.AppCompatActivity;

import com.example.fittrack.databinding.ActivityIntroBinding;

public class IntroActivity extends AppCompatActivity {
ActivityIntroBinding binding;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding=ActivityIntroBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        getWindow().setFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS,WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS);

        binding.startBtn.setOnClickListener(view -> startActivity(new Intent(IntroActivity.this,MainActivity.class)));
    }
}

File: app/src/main/java/com/example/fittrack/Activity/IntroActivity.kt
package com.example.fittrack.Activity

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.view.WindowManager
import androidx.appcompat.app.AppCompatActivity
import com.example.fittrack.databinding.ActivityIntroBinding

class IntroActivity : AppCompatActivity() {
    var binding: ActivityIntroBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityIntroBinding.inflate(layoutInflater)
        setContentView(binding!!.root)

        window.setFlags(
            WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS,
            WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS
        )

        binding!!.startBtn.setOnClickListener { view: View? ->
            startActivity(
                Intent(
                    this@IntroActivity, MainActivity::class.java
                )
            )
        }
    }
}